Resources/Prototypes/Loadouts/role_loadouts.yml

```diff
@@ -346,10 +346,10 @@
   groups:
   - MedicalMask
   - ChemistJumpsuit
-  - MedicalGloves
+  - ChemistGloves # DeltaV
   - ChemistBackpack
   - ChemistOuterClothing
-  - MedicalShoes
+  - ChemistShoes # DeltaV
```

Comment:
This needs to be commented out, and replaced with Chemist gloves *or* this needs to have a comment saying "Delta-V : Replaces MedicalGloves/Shoes with CHemistGloves/Shoes"